test.js

```diff
@@ -1,100 +1,41 @@
 var expect = require('chai').expect;
-var Bitmap_Transformer = require('./Bitmap_Transformer.js');
+var BitmapTransformer = require('./Bitmap_Transformer.js').BitmapTransformer;
 var fs = require('fs');
 
+var FitnessEmoji = new BitmapTransformer();
```

Comment:
for instance variables, use camel case: `fitnessEmoji`. In javascript, only classes (constructor functions) are pascal case.